### PR TITLE
Create Collection component

### PR DESCRIPTION
**Pull request by GuRuGuMaWaRu**
1) Created a basic Collection component and wrote some cases for it in React Storybook.
2) Changed Card component styles a little in order to use in React Storybook with Collection component.

### CODE REVIEW

File: slinker_web/src/client/scss/components/Collection.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    text-indent: -1000rem;
+    text-transform: none;
+    background-size: 1.2rem 1.2rem;
+    background-color:#000;          
+    outline: none;
+    cursor: pointer;
+
+    &__expand {
+      /* https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images */
+      -webkit-mask:  url('/images/expand-icon.svg') no-repeat 50% 50%;
```
Comment (scopevale) [inline slinker_web/src/client/scss/components/Collection.scss]: This doesn't work at all in IE 11 or Edge 14/15

File: slinker_web/src/client/scss/components/Collection.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      mask: url('/images/expand-icon.svg') no-repeat 50% 50%;  
+
+      &:hover {
+        background-color: red;
+      }
+    }
+
+    &__collapse {
+      /* https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images */
+      -webkit-mask:  url('/images/collapse-icon.svg') no-repeat 50% 50%;
```
Comment (scopevale) [inline slinker_web/src/client/scss/components/Collection.scss]: doesn't work in IE 11 or Edge 14/15

**Review by scopevale — CHANGES_REQUESTED** [review]

File: slinker_web/src/client/scss/components/Collection.scss (hunk 2 above)
Comment (GuRuGuMaWaRu) [reply slinker_web/src/client/scss/components/Collection.scss]: OK, I'll look for something different then

**Comment by GuRuGuMaWaRu** [comment]
Tried 'filters', but they didn't work for me, so instead used another method. Now on hover I replace current (black) background SVGs with the other (red).

**Review by scopevale — APPROVED** [review]

**Pull request merged**